### PR TITLE
Task 738 from perspectrum

### PR DESCRIPTION
**Pull request by Palipoor**

### CODE REVIEW

**Comment by swarooprm** [comment]
Task 685?

**Comment by Palipoor** [comment]
> Task 685?

Task 685 seems to be here #308

**Comment by swarooprm** [comment]
Great! Thanks for confirming.

**Comment by swarooprm** [comment]
@Palipoor, this task looks good. One comment: before you write the perspective, there should be an identifier such as '.' , right now it appears as it perspective is part of the same sentence.

I assume labels are balanced here, if not please balance them.

**Comment by danyaljj** [comment]
@schen149 if you have thoughts or suggestions, please let @Palipoor know, thanks!

**Comment by Palipoor** [comment]
This will solve issue #138

File: tasks/task738_perspectrum_classification.json [hunk 1]
```diff
@@ -0,0 +1,5301 @@
...
+    "Definition": "In this task you will be given a claim and a perspective. You should determine whether that perspective supports or undermines the claim. If the perspective could possibly convince someone with different view, it is supporting, otherwise it is undermining.",
+    "Positive Examples": [
+        {
+            "input": "claim: Music containing lyrics that glorify violent and criminal lifestyles should be banned.\n perspective: hip hop artists have a right to free speech",
+            "output": "undermine",
+            "explanation": "Musicians having a right to free speech clearly undermines the claim for putting constraints on their lyrics."
+        },
+        {
+            "input": "claim: Abolish the US Electoral College.\n perspective: The electoral college weakens incentives for voting and party building.",
+            "output": "supoprt",
```
Comment (danyaljj) [inline tasks/task738_perspectrum_classification.json]: support?

**Comment by danyaljj** [comment]
Except for one typo, all looks good to me.

**Comment by Palipoor** [comment]
Solved! Thanks.

**Pull request merged**